### PR TITLE
[Magiclysm] New Spell: Nature's Commune

### PR DESCRIPTION
**Pull request by Termineitor244**
#### Summary
Mods "Adds a new druidic spell giving the effects of Animal Empathy and a little health"

#### Purpose of change

Druids are all about their relation with nature, but they currently don't have any means to be in peace with that same nature (fauna), so why don't give them a spell with that effect?

#### Describe the solution

A new druidic spell "Nature's Commune", that makes you enter a magical state of communion with the nature around you, calming the natural animals and making you a little healthier, since you are "embraced" by the mother nature, the health effect is mainly of real (immediate) health, so given enough time without another commune, your body will tend towards its old, unhealthy state.

#### Describe alternatives you've considered

Two different spells, or a healing/stamina regeneration effect.

#### Testing

Made the json changes, tested, they give the health in the proportions I want, I then compiled the game, and the effect in wildlife works as it should, a bunch of rabbits stay in my vicinity if I'm under the effect of the spell, and run away the moment the spell duration ends.

#### Additional context

This was originally a regeneration buff spell, but I discarded that idea and ended with this (in my opinion) better and needed alternative.

This is a draft because I still have to add the spell to books that seem appropriate, but if anyone wants to make changes to the spell itself, I'm open to suggestions.

Edit: Added the spell to the basic druidic book, to a new scroll, and to the Ecoterrorist profession (seems appropriate).

### CODE REVIEW

File: data/mods/Magiclysm/Spells/druid.json [hunk 1]
```diff
@@ -396,5 +396,27 @@
     "shape": "blast",
     "effect": "summon",
     "effect_str": "mon_wolf"
+  },
+  {
+    "id": "druid_natures_commune",
+    "type": "SPELL",
+    "name": "Nature's Commune",
+    "description": "Concentrate in the nature around you and commune with it, improving your health and calming the nearby wildlife.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/Spells/druid.json:404]:
```suggestion
    "description": "You concentrate on the nature around you and commune with it, improving your health and calming the nearby wildlife.",
```

File: data/mods/Magiclysm/effects/effects.json [hunk 2]
```diff
@@ -300,5 +300,17 @@
     "int_add_val": 1,
     "base_mods": { "speed_mod": [ -25 ] },
     "scaling_mods": { "speed_mod": [ -5 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "natures_commune",
+    "name": [ "Nature's Commune" ],
+    "desc": [ "You can feel Mother Nature welcoming you, revitalizing your body and calming the natural fauna in your way." ],
+    "apply_message": "A warmth embrace surrounds you, like that of a mother with her child.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/effects/effects.json:309]:
```suggestion
    "desc": [ "You can feel Mother Nature welcoming you, revitalizing your body and calming the natural fauna around you." ],
    "apply_message": "A warm embrace surrounds you, like that of a mother with her child.",
```

File: data/mods/Magiclysm/items/spell_scrolls.json [hunk 3]
```diff
@@ -1026,5 +1026,13 @@
     "name": { "str": "Scroll of freezing touch", "str_pl": "Scrolls of freezing touch" },
     "description": "Your hands freeze anything they touch at temperatures so cold it slows down your foes.",
     "use_action": { "type": "learn_spell", "spells": [ "freezing_touch" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_natures_commune",
+    "name": { "str": "Scroll of Nature's Commune", "str_pl": "Scrolls of Nature's Commune" },
+    "description": "Your connection with nature allows you to become one with it, revitalize your body while you enter in communion with all that surrounds you.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/items/spell_scrolls.json:1035]:
```suggestion
    "description": "Your connection with nature allows you to become one with it; you revitalize your body while in communion with the flora and fauna around you.",
```

**Review by actual-nh — COMMENTED** [review]
Nice! (@BrettDong: Unfortunately needs a String Freeze label.)

**Comment by Termineitor244** [comment]
@actual-nh Thanks for the help with the grammar and syntaxis!

**Comment by KorGgenT** [comment]
ran out of time to check this PR this morning. what i need to check before i can make a determination on the balance (or maybe you can tell me) is how much total hp you regenerate in the duration of the spell.

**Pull request merged**